website/sidebars.json

```diff
@@ -77,6 +77,7 @@
           "querying/sql-scalar",
           "querying/sql-aggregations",
           "querying/sql-multivalue-string-functions",
+          "querying/nested-columns",
```

Comment:
i forgot to add sql-json-functions here in https://github.com/apache/druid/pull/12922, could you add it?

Also, i think maybe nested-columns should live in "concepts" section instead?

Reply:
Done